### PR TITLE
#514 Increase simulation time and event time when SVC are used

### LINKED ISSUE #514: [SVC] update default stop time and time of event in presence of a SVC

### PR DESCRIPTION
**Pull request by rosiereflo**
closes #514

**Checklist before requesting a review**

*use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes*

- [x] unit tests and non regression tests were added (update of inputs, outputs, SA, algos)
- [ ] main documentation was updated (update of input/output file)
- [x] the corresponding milestone was added in the ticket and in this PR

### CODE REVIEW

**Review by cossartque — APPROVED** [review]

**Comment by cossartque** [comment]
> image
> 
> For now I have put a duration of 2000s and an event at 500s what do you think? (Only if nothing was specified in the input configuration file)

The duration of 2000s is for N and N-1 ?

**Comment by rosiereflo** [comment]
> 
> The duration of 2000s is for N and N-1 ?

Yes this is the current (safe) choice to make sure the SVC is stabilized. This is just a first attempt, we can refine it later on

File: sources/Context.h [hunk 1]
```diff
@@ -50,11 +50,11 @@ namespace dfl {
...
   using ProcessNodeCallBackMainComponent =
-      std::function<void(const std::shared_ptr<inputs::Node>&,
-                         std::shared_ptr<dfl::algo::AlgorithmsResults>&)>;  ///< Callback for node algorithm on main topological island
-                                                                            /**
-   * @brief Context definition
-   */
+      std::function<void(const std::shared_ptr<inputs::Node> &,
+                         std::shared_ptr<dfl::algo::AlgorithmsResults> &)>;  ///< Callback for node algorithm on main topological island
+                                                                             /**
+                                                                              * @brief Context definition
```
Comment (marcochiaramello) [inline sources/Context.h:56]: comment is shift

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: sources/Context.h (hunk 1 above)
Comment (rosiereflo) [reply sources/Context.h]: done

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**